castor/castor/urls.py

```diff
@@ -0,0 +1,53 @@
+"""castor URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/1.10/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.conf.urls import url, include
+    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
+from django.conf.urls import include
+from django.conf.urls import url
+from django.contrib import admin
+from django.contrib.auth.views import logout_then_login
+from rest_framework import routers
+
+from docker_events.views import DockerEventViewSet
+from docker_servers.views import DockerServerViewSet
+from web.views import home
+from web.views import signin
+from web.views import webhooks
+from web.views import webhook
+from webhooks.views import DeliveryViewSet
+from webhooks.views import WebHookDeliveryViewSet
+from webhooks.views import WebHookViewSet
+
+
+router = routers.DefaultRouter()
```

Comment:
We could include these urls in a `webhooks/urls.py` file which would expose a `urlpatterns` parameter and then import all these URLs. This will keep applications more contained.

See an example [here](https://github.com/sourcelair/stolosd/blob/master/projects/urls.py) and [here](https://github.com/sourcelair/stolosd/blob/master/stolosd/urls.py#L32) of how this could work.